docs/usage/pod-security.md

```diff
@@ -8,8 +8,44 @@ admissionPlugins:
 - name: PodSecurityPolicy
   disabled: true
 ```
-in `kubernetes.kubeAPIServer.AdmissionPlugins` field in the shoot spec. Please refer the example shoot manifest [here](../../example/90-shoot.yaml).
+in `kubernetes.kubeAPIServer.admissionPlugins` field in the shoot spec. Please refer the example shoot manifest [here](../../example/90-shoot.yaml).
 
 Only if this field is set, the cluster can be upgraded to kubernetes `v1.25`.
 
 <strong>Note:</strong> You should disable the admission plugin and wait till Gardener finish at least one shoot reconciliation before upgrading to `v1.25`. This is to make sure all the `PodSecurityPolicy` related resources deployed by Gardener are cleaned up.
+
+## Admission configuration for the `PodSecurity` admission plugin
+
+If `.spec.kubernetes.allowPrivilegedContainers` is set to false in the Shoot spec, and the `PodSecurityPolicy` admission plugin is disabled, an admission configuration for the `PodSecurity` admission plugin with `restricted` level as default is applied. If you wish to add your own custom configuration, you can do so in the Shoot spec under `.spec.kubernetes.kubeAPIServer.admissionPlugins` by adding:
```

Comment:
```suggestion
If `.spec.kubernetes.allowPrivilegedContainers` is set to `false` in the Shoot spec and the `PodSecurityPolicy` admission plugin is disabled, an admission configuration for the `PodSecurity` admission plugin with `restricted` level as default is applied. If you wish to add your own custom configuration, you can do so in the Shoot spec under `.spec.kubernetes.kubeAPIServer.admissionPlugins` by adding:
```

Reply:
You could make more clear that end-users cannot overwrite the defaults when `allowPrivilegedContainers=false`.